docs/001_develop/03_client-capabilities/009_presentation/banner.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: 'Banner'
+sidebar_label: 'Banner'
+id: client-presentation-banner
+keywords: [presentation, banner, personalisation, user, image]
+tags:
+- presentation
+- banner
+- personalisation
+- user
+- image
+sidebar_position: 9
+---
+
+import BannerDemo from './examples/banner.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Banner
+
+Use cases:
+* Highlighting an item.
+* Flagging an item as part of a category.
+* Representing a value of unread messages.
+
+## Example
+
+<BannerDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+  <TabItem value="genesis">
+
+    Declaration
+    ```html
+    <rapid-banner></rapid-banner>
+    ```
+
+    Usage
+    ```typescript
+    @customElement({
+      name: 'my-element',
+      template: html`
+        <rapid-banner
+          ${ref('bannerRef')}
+          @click=${x => x.handleClick()}>
+        </rapid-banner>
```

Comment:
Can you put the same information you've put in the react and angular example in the content slot here too please